src/main/java/ua/lviv/iot/lab4/models/Сalculators.java

```diff
@@ -0,0 +1,45 @@
+package ua.lviv.iot.lab4.models;
+
+public class �alculators extends OfficeSupplies {
```

Comment:
Might sound strange, but why Ñ? I believe you have a typo.

Reply:
Will write it here so we don't miss it. **Class names should be singular nouns:** Calculator, OfficeSupply etc.